packages/block-directory/src/store/utils/get-plugin-url.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Get the plugin's direct API link out of a block-directory response.
+ *
+ * @param {Object} block The block object
+ *
+ * @return {string} The plugin URL, if exists.
+ */
+export default function getPluginUrl( block ) {
+	if ( ! block ) {
+		return false;
+	}
+	const link = block.links[ 'wp:plugin' ] || block.links.self;
+	if ( link && link.length ) {
+		return link[ 0 ].href;
+	}
+	return false;
```

Comment:
Not incredibly important but `@wordpress/url`'s `getPath` returns `undefined` (which we use in `controls.js`). 

I wonder if it makes sense to follow that convention.